src/main/java/edu/ksu/canvas/attendance/controller/CourseConfigurationController.java

```diff
@@ -83,6 +83,7 @@ public ModelAndView classSetup(@PathVariable String sectionId,
         CourseConfigurationForm courseConfigurationForm = new CourseConfigurationForm();
         courseService.loadIntoForm(courseConfigurationForm, selectedSection.getCanvasCourseId());
         sectionService.loadIntoForm(courseConfigurationForm, selectedSection.getCanvasCourseId());
+        courseConfigurationForm.setAllSections(sectionService.getSectionByCanvasCourseId(selectedSection.getCanvasCourseId()));
         page.addObject("courseConfigurationForm", courseConfigurationForm);
         page.addObject("selectedSectionId", selectedSection.getCanvasSectionId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "selectedSectionId" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)